velox/type/Type.h

```diff
@@ -411,6 +411,10 @@ class Type : public Tree<const std::shared_ptr<const Type>>,
   }
 
   virtual bool isFixedWidth() const = 0;
+  // used in FixedSizeArrayType to return the width constraint of the type
```

Comment:
nit: add blank line in between
nit2: comments are usually full sentences, with capital letter and punctuation.

check the coding style guide:

https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md